backend.native/tests/build.gradle

```diff
@@ -931,7 +939,6 @@ standaloneTest("cleaner_in_main_without_checker") {
 standaloneTest("cleaner_leak_release") {
     enabled = !project.globalTestArgs.contains('-g') && (project.testTarget != 'wasm32') // Cleaners need workers
     source = "runtime/basic/cleaner_leak.kt"
-    goldValue = ""
```

Comment:
Is removing `goldValue` still required?

Reply:
Actually not, I'm going to rebase and squash some commits. Also this setting goldValue is not needed as this is an optional nullable parameter.

Reply:
> Also this setting goldValue is not needed as this is an optional nullable parameter.

`goldValue = ""` isn't the same as `goldValue = null`.
The former checks that output is empty (and this might be the original intention), the latter doesn't.

Reply:
> and this might be the original intention

It is.